src/components/editions/header.tsx

```diff
@@ -55,7 +55,7 @@ const AnalysisHeader: FC<HeaderProps> = ({ item }) => (
 );
 
 const InterviewHeader: FC<HeaderProps> = ({ item }) => (
-	<header css={headerStyles}>
```

Comment:
Just wonder if should rename the old `headerStyles` css to `headerPadding` or something to make clear what its doing?

Reply:
or `sidePadding` even?

Reply:
Yeah I agree assuming we are done adding default header styles? Otherwise headerPadding might be a bit narrow.

Reply:
Good point - I'd imagine from here things are only going to get less generic? Thats said for the sake of not undoing things, If we leave it as for now maybe then and we can change down the road if it makes sense?

Reply:
Gotcha